packages/libs/client/src/integration-tests/helpers/account/fund-account.ts

```diff
@@ -0,0 +1,56 @@
+import { ChainId, IPactDecimal } from '@kadena/types';
+
+import { Pact, readKeyset } from '../../../index';
+import { NetworkId } from '../../support/enums';
+import { sender00Account } from '../../test-data/accounts';
+import { listen, preflight, submit } from '../client';
+import { signByKeyPair } from '../transactions/sign-transaction';
+
+export async function fundAccount(
+  receiver: string,
+  receiverKey: string,
+  amount: IPactDecimal,
+  chain: ChainId,
+): Promise<string | undefined> {
+  const transaction = Pact.builder
+    .execution(
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (Pact.modules as any).coin['transfer-create'](
+        sender00Account.account,
+        receiver,
+        readKeyset('ks'),
+        amount,
+      ),
```

Comment:
I tried making the typings available by running the first mentioned script (I had to slightly update it) this seems to generate typings but then using pact.modules.coin doesn't appear to work. I might need a tiny little bit of help getting that to work.

Reply:
We can't generate types for the contract here. The problem is that this file is in the client package, and we use interface merging for Pact types. This will break types in the pact.ts file itself. If we can somehow isolate this section, maybe with a separate tsconfig file, we might be able to fix it.